community/technical-team.md

```diff
@@ -4,16 +4,19 @@ title: "Technical team"
 ---
 # Technical team
 
-The Technical Team consists of interested developers and two Board-appointed Co-Chairs. The Co-Chairs act as administrative chairs of the Technical Team, e.g. calling meetings and coordinating development efforts. They also have administrative privileges for the MEI repositories. The MEI Technical Team holds regular virtual and in-person meetings, maintains the MEI repository on GitHub ([https://github.com/music-encoding](https://github.com/music-encoding)), prepares new releases of MEI by reviewing and implementing revisions and features requested by the community, and supports the wider MEI community with their technical expertise.
+The Technical Team consists of interested developers and two Board-appointed Co-Chairs. The Co-Chairs act as administrative chairs of the Technical Team, e.g., calling meetings and coordinating development efforts. They also have administrative privileges for the MEI repositories. The MEI Technical Team holds regular virtual and in-person meetings, maintains the MEI repository on GitHub ([https://github.com/music-encoding](https://github.com/music-encoding)), prepares new releases of MEI by reviewing and implementing revisions and features requested by the community, and supports the wider MEI community with their technical expertise.
 
 * **Benjamin W. Bohl (Technical Co-Chair)**, Institut für Musikwissenschaft, Goethe Universität Frankfurt, Frankfurt am Main and Musikwissenschaftliches Seminar Detmold/Paderborn
-* **Johannes Kepper (Technical Co-Chair)**, Musikwissenschaftliches Seminar Detmold/Paderborn
-* **Andrew Hankinson**, RISM Digital Center, Bern
-* **Perry Roland**, Alderman Library, University of Virginia, Charlottesville
+* **Sophia Dörner**, Humboldt-Universität zu Berlin
 * **Ichiro Fujinaga**, McGill University, Montréal
+* **Andrew Hankinson**, RISM Digital Center, Bern
+* **Johannes Kepper**, Musikwissenschaftliches Seminar Detmold/Paderborn
+* **Stefan Münnich (Technical Co-Chair)**, Anton Webern Gesamtausgabe, University of Basel
```

Comment:
I know we're trying to keep things alphabetical, but it would be great if the co-chairs were both listed first. :)

Reply:
I'm fine with alphabetical order but maybe that's just because I'm in a luck position with my initials 🥇  ;-)

Just kidding, sounds like a good idea to have the Administrative Co-Chairs at the top.